CHANGELOG.md

```diff
@@ -15,6 +15,8 @@ All notable changes to this project will be documented in this file.
 
 * Exception message in AESCipher (thrown if key has unexpected length)
 * Install target of documentation
+* Add error detection to utility::fromHex (This changes the behaviour of fromHex as it will throw
```

Comment:
I think this should be in the "changed" section, since it's a change of behavior, not necessarily a bugfix.